Sources/Commands/Snippets/Cards/SnippetCard.swift

```diff
@@ -39,20 +40,35 @@ struct SnippetCard: Card {
     var swiftCommandState: SwiftCommandState
 
     func render() -> String {
-        var rendered = colorized {
+        let isColorized: Bool = swiftCommandState.options.logging.colorDiagnostics
+        var rendered = isColorized ? colorized {
             brightYellow {
                 "# "
                 snippet.name
             }
             "\n\n"
         }.terminalString()
+            :
+            plain {
```

Comment:
question: Out of curiosity, why do we need two nested `plain` here?  can this something like this

```
plain {
    "# "
    snippet.name
   "\n\n"
}
```